README.md

```diff
@@ -4,6 +4,30 @@
 This tool allows you to mirror RPM repositories in your own private network.
 Organization (mirroring) credentials are required to mirror SUSE repositories.
 
+## Installation and configuration
+
+RMT currently gets build for these distributions: `SLE_12_SP2`, `SLE_12_SP3`, `openSUSE_Leap_42.2`, `openSUSE_Leap_42.3`, `openSUSE_Tumbleweed`.
+To add the repository, please call: (replace `<dist>` with your distribution)
```

Comment:
Please remove all `please`s from this and all other files :)
Doc team told us we should not use that word.